### PR TITLE
Contact form branch

### PR DESCRIPTION
**Pull request by elarhadu**
This is a simple portfolio (mobile version) built with Figma template 5. It contains the contact form. It was built with HTML and CSS, using flexbox and a CSS grid. The main objective of this project was to:

- Create forms with HTML5 validations.
- Understand the importance of UX.
- Understand how to parse a Figma design to create a UI.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -835,3 +849,251 @@ p {
...
+
+.end {
+  position: relative;
+  width: 74px;
+  height: 74.02px;
+  right: -19px;
+  top: 3793px;
+}
+
+.next-step {
```
Comment (christianbiring1) [inline style.css:862]: - [ ] Kindly make sure your page fits the screen size on the mobile version. We do not need to scroll to see the entire page width. consider fixing that.
Your design :point_down: 
Screenshot from 2022-11-01 19-07-57

Figma template :point_down: 
image

File: index.html [hunk 2]
```diff
@@ -138,5 +138,48 @@ <h1 class="me">About</h1>
     </div>
     </article>
     </section>
+    <footer>
```
Comment (christianbiring1) [inline index.html]: - [ ] `footer` doesn't have any semantic meaning here, kindly make sure you remove that.
`You can check` [here](https://www.w3schools.com/tags/tag_footer.asp) `when to use footer`

File: index.html [hunk 3]
```diff
@@ -138,5 +138,48 @@ <h1 class="me">About</h1>
...
+            <img src="images/logo.png" alt="logo" class="end-logo">
+            <h2 class="next-step">Get started</h2>
+            <p class="conclusion">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+          </nav>
+          <div class="form">
+        <form action="https://formspree.io/f/xzbwkgkl" method="POST" class="follow-up">
+            <label class="your-name">Your name</label>
+           <input class="name" type="text" name="name" placeholder="Anastasiia" required="" maxlength="30" ><br>
+           <label class="your-email">Your email</label>
+          <input class="email" type="email" name="Email address" placeholder="" required="" autofocus><br>
```
Comment (christianbiring1) [inline index.html]: [OPTIONAL]
- [ ] You may consider removing the value `""` on the attribute `required`

File: index.html [hunk 4]
```diff
@@ -138,5 +138,48 @@ <h1 class="me">About</h1>
     </div>
     </article>
```
Comment (christianbiring1) [inline index.html:139]: [OPTIONAL]
- [ ] Kindly always make sure the previous work (`milestone`) is included in your newly opened pull request

File: style.css [hunk 5]
```diff
@@ -835,3 +849,251 @@ p {
...
+}
+
+.form {
+  position: relative;
+  width: 350px;
+  top: -85px;
+  left: -7px;
+  display: grid;
+  justify-content: center;
+  align-items: center;
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure you use the flexbox to place your elements on this specific section as required

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @elarhadu @edenlisk , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good UI :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] `I highly recommend you implement them for they will take your project to the next level` :rocket: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (elarhadu) [reply style.css:862]: Hi @christianbiring1, can you please review this comment again? Because the above screenshot provided isn't my project.

**Review by OyePriscilla — APPROVED** [review]
## Hi @elarhadu   :smiley: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 All linters checks are passed ✔️
 Descriptive PR title and summary ✔️ 
 Good UI ✔️ 
Descriptive commit messages ✔️ 

## Optional suggestions
[optional] Kindly take note that your header section is missing, This PR suppose to be a continuation of your work. Kindly add the header section. 👍 

crr12


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**